openmdao/utils/variable_table.py

```diff
@@ -212,6 +214,7 @@ def _write_variable(out_stream, row, column_names, var_dict, print_arrays):
             out_stream.write("{}  {}:\n".format(
                 left_column_width * ' ', column_name))
             out_str = str(var_dict[column_name])
+            out_str = pprint.pformat(var_dict[column_name])
```

Comment:
looks like you meant to delete/replace the previous line...

Reply:
Fixed.